Gemfile

```diff
@@ -48,6 +48,10 @@ gem 'jquery-rails', '~> 4.3', '>= 4.3.5'
 
 gem 'chartkick', '~> 3.3', '>= 3.3.1'
 
+gem 'stub_env', '~> 1.0'
+
+gem 'webmock', '~> 3.8'
+
```

Comment:
I think these gems should be added within the `test` group

Reply:
My bad, you're right!